src/containers/reports/ReportsSagas.js

```diff
@@ -340,26 +290,43 @@ function* updateReportWorker(action :SequenceAction) :Generator<*, *, *> {
     return;
   }
 
-  const entityKeyId :string = value.entityKeyId;
-  const entitySetId :string = value.entitySetId;
-  const reportEdits :Map<string, List<any>> = value.reportEdits;
+  const {
+    entityKeyId,
+    formData,
+  } = value;
 
   try {
     yield put(updateReport.request(action.id, value));
 
     const edm :Map<*, *> = yield select(state => state.get('edm'));
-    const entityData :Map<string, List<any>> = Map().withMutations((map :Map<string, List<any>>) => {
-      reportEdits.forEach((data :List<any>, fqn :FullyQualifiedName) => {
-        const propertyTypeId :string = edm.getIn(['fqnToIdMap', fqn]);
-        map.setIn([entityKeyId, propertyTypeId], data);
-      });
+    const app = yield select(state => state.getIn(['app'], Map()));
```

Comment:
unimportant but you don't need to use getIn here -- seems a teensy bit cleaner to use .get and avoid the extra brackets

Reply:
tyty